<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanGroupLayout</class>
 <widget class="QWidget" name="ScanGroupLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_5" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_9" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_6" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_10" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_3" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_7" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="QWidget" name="widget_11" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_4" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_8" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
